reactioncommerce/reaction-next-starterkit
# [WIP] feat: Media gallery

Issue: #23
Impact: **minor**
Type: **feat**

## Issue

Create a media gallery component that has a featured image, and a list of clickable images in a list below.

## Solution

- Create a `MediaGallery` component that contains the layout and handles state
- Create a `MediaGalleryItem` component that can serve as the featured image as well as sub-images

## Testing

- Start the app and go to `http://localhost:4000/product/example-product`
- See the Media gallery to the left with sample images and data
- Clicking on a sub-item should change the featured image
